docs/docs/end-to-end-testing.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "End-to-end testing"
+---
+
+[Cypress](https://www.cypress.io/) is one of the options when it comes to end-to-end testing. Setting up and writing tests in Cypress is developer-friendly and easy to learn. You'll profit from the time travel feature or the extensive debugging capabilities with Chrome DevTools. Of course you can also use it with Gatsby and this guide will explain how.
+
+In order to run Gatsby's development server and Cypress at the same time you'll use the little helper [start-server-and-test](https://github.com/bahmutov/start-server-and-test). If you're already using [react-testing-library](docs/testing-react-components) you might want to install `cypress-testing-library`, too. Install the following packages to your `devDependencies`:
```

Comment:
Why would someone want to use both `react-testing-library` and `cypress-testing-library`?

Reply:
Yeah, the sentence should be more clear:
„If you already use react-testing-library (RTL) for unit testing you might ...“
+ a little explanation why

The reason is: If you use RTL which has ways of selecting elements, you can use them in Cypress too (with the cypress thingy)